src/convertToObject.js

```diff
@@ -5,8 +5,30 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const delEmptyLine = sourceString.replace(/\n/g, '');
+
+  const commandsArr = delEmptyLine.split(';');
+
+  const commandsArrFinal = commandsArr.filter((str) => str.trim());
+
+  const resultObject = {};
+
+  for (let i = 0; i < commandsArrFinal.length; i++) {
+    const commandArr = commandsArrFinal[i].split(':');
+
+    const key = commandArr[0].trim();
+    const value = commandArr[1].trim();
+
+    resultObject[key] = value.replace(/,/g, ',\n');
```

Comment:
[CODE KNOWLEDGE] - It is not required to handle cases with multiple colons within the style string's property values. The replacement of commas with a newline is unnecessary and not part of the task requirements.